GoogleForCreators/web-stories-wp
# Archive: Redirect default archive page to custom one

## Context



When users choose a custom page as the Web Stories archive page, we probably want to add a redirect from `/web-stories/` to the custom archive page for UX reasons.
Probably also from `/web-stories/feed/` to `/new/custom/path/feed/`

## Summary

Adds a redirect from the default archive page to the custom one if one is set up.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Enable, archive page experiment. 
2. Go to Dashboard > Setting. 
3. Change Stories Archives to `custom`
4. Select a page from dropdown, ( you will need at least one page ). 
5. Go to `<site_url>/web-stories`. Browser should redirect to path of selected page. 
6. Go to a single stories url, see it not redirect. 
7. Go to Dashboard > Setting. 
8. Set archive back to default. 
9. Go to `<site_url>/web-stories`. Browser shouldn't redirect to path of selected page. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9100
